hypoddpy/hypodd_relocator.py

```diff
@@ -384,9 +384,9 @@ def _write_catalog_input_file(self, phase_weighting=None):
                         station_id=pick["station_id"])
                     self.log(msg, level="warning")
                     continue
-                weight = phase_weighting(pick['station_id'], pick['phase'],
+                weight = phase_weighting((pick['station_id'], pick['phase'],
                                          pick['pick_time'],
-                                         pick['pick_time_error'])
+                                         pick['pick_time_error']))
```

Comment:
Why did you change this? The default unit phase weighting (https://github.com/krischer/hypoDDpy/blob/766d92ae4ced9c111ebb3217e9bd0defce095604/hypoddpy/hypodd_relocator.py#L76) expects 4 parameters.

With your change its suddenly a single tuple parameter.

If you use a custom phase weighting function, please just make it take four parameters.